AzureIoTHub.agent.lib.nut

```diff
@@ -36,12 +37,40 @@
 
 /// Azure AzureIoTHub library
 
-const AZURE_HTTP_API_VERSION = "2016-11-14";
+const AZURE_API_VERSION = "2016-11-14";
 const AZURE_CLAIM_BASED_SECURITY_PATH = "$cbs";
 
+const AZURE_IOT_CLIENT_STATE_DISCONNECTED           = 1;
+const AZURE_IOT_CLIENT_STATE_DISCONNECTING          = 2;
+const AZURE_IOT_CLIENT_STATE_CONNECTED              = 4;
+const AZURE_IOT_CLIENT_STATE_CONNECTING             = 8;
+const AZURE_IOT_CLIENT_STATE_ENABLING_MSG           = 1024;
+const AZURE_IOT_CLIENT_STATE_DISABLING_MSG          = 2048;
+const AZURE_IOT_CLIENT_STATE_ENABLING_TWIN          = 4096;
+const AZURE_IOT_CLIENT_STATE_DISABLING_TWIN         = 8192;
+const AZURE_IOT_CLIENT_STATE_ENABLING_DMETHOD       = 16384;
+const AZURE_IOT_CLIENT_STATE_DISABLING_DMETHOD      = 32768;
+
+const AZURE_IOT_CLIENT_ERROR_NOT_CONNECTED          = 1000;
+const AZURE_IOT_CLIENT_ERROR_ALREADY_CONNECTED      = 1001;
+const AZURE_IOT_CLIENT_ERROR_NOT_ENABLED            = 1002;
+const AZURE_IOT_CLIENT_ERROR_ALREADY_ENABLED        = 1003;
+const AZURE_IOT_CLIENT_ERROR_OP_NOT_ALLOWED_NOW     = 1004;
```

Comment:
just wondering why not to use 429 instead?

Reply:
Done